package.json

```diff
@@ -24,12 +24,13 @@
     "clean:tslint": "rimraf tslint-rules/*.js",
     "compile:tslint": "tsc -p tslint-rules",
     "compile:script": "tsc -p script",
-    "lint": "yarn compile:tslint && yarn compile:script && yarn tslint && yarn eslint-check && yarn eslint",
-    "lint:fix": "yarn compile:tslint && yarn tslint:fix && yarn eslint:fix",
+    "lint": "yarn lint:src && yarn lint:prettier",
+    "lint:fix": "yarn lint:src:fix && yarn lint:prettier --write",
+    "lint:prettier": "prettier '**/*.md' '**/*.scss' --list-different",
```

Comment:
💭  while this command here is nice and succinct about what needs to be fixed, someone first encountering this for the first time doesn't really get much context:

```shellsession
~/src/desktop/ yarn lint
yarn run v1.3.2
$ yarn lint:src && yarn lint:prettier
$ yarn tslint && yarn eslint-check && yarn eslint
$ tslint ./{script,tslint-rules}/*.ts ./app/{src,typings,test}/**/*.{ts,tsx}
$ eslint --print-config .eslintrc.* | eslint-config-prettier-check
No rules that are unnecessary or conflict with Prettier were found.
$ ts-node script/eslint.ts

$ prettier '**/*.md' '**/*.scss' --list-different
CONTRIBUTING.md
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

We often have to go in and explain what's happening here when a PR is opened and fails CI, but maybe we can do better to reduce that work:

 - some documentation around "things you should do before submitting a PR"
 - wrapping this in a custom script to handle the exit code on error, display some context around this output and indicate what should be done next

Or maybe a combination of the two?

Reply:
👍 I’ll look into making putting a message if it failed.